.github/workflows/ci.yml

```diff
@@ -0,0 +1,18 @@
+name: linting
+on: [push, pull_request]
+jobs:
+  test:
+    runs-on: macos-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: terraform fmt
+        run: |
+          brew install tfenv
+          tfenv install
+          tfenv install 0.13.0-beta3
+          tfenv use 0.13.0-beta3
```

Comment:
```suggestion
```

`tfenv install` pulls from the repo's `.terraform-version` which should be in the repository root (or per-project if it needs to be different).